adafruit_display_text/bitmap_label.py

```diff
@@ -111,136 +111,220 @@ def __init__(
         anchor_point=None,
         anchored_position=None,
         save_text=True,  # can reduce memory use if save_text = False
-        **kwargs
+        scale=1,
     ):
 
-        if text == "":
-            raise RuntimeError(
-                "Please provide text string, or use label.py for mutable text"
-            )
+        # instance the Group
+        # self Group will contain a single local_group which contains one TileGrid which contains
+        # the text bitmap
+        super().__init__(
+            max_size=1, x=x, y=y,
+        )  # this will include any arguments, including scale
```

Comment:
```suggestion
        )
```

This is no longer true because **kwargs was removed. Are you sure you want to remove it? Doing so prevent passing other Group arguments.

Reply:
displayio.Group has only four parameters (max_size, x, y and scale).  But I added the `**kwargs` passing back in case Group is updated in the future to handle more parameters.  I made this change in both `bitmap_label.py` and `label.py`.